src/app/heroes/heroes.component.ts

```diff
@@ -17,7 +17,7 @@ export class HeroesComponent implements OnInit {
   
   constructor(private heroService: HeroService) {}
 
-  ngOnInit(): {
```

Comment:
A.g.v die silly strictness van die comment hieronder, het typescript verwag dat jy 'n return type specify vir die (lifecycle) method. Omdat dit nog nie iets is wat jy hoef te weet nie, dink ek het jy dalk probeer om dit te fix deur suggestions in die editor of op die internet te volg. Dit was amper daar, het net nog 'n stukkie gekort. Hierdie sou reg wees:

`ngOnInit(): void {`

Wat betekend ngOnInit het niks om te return nie.